Python/peephole.c

```diff
@@ -477,10 +500,12 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
                 break;
         }
         nexti = i - op_start + 1;
```

Comment:
It may be better to introduce a new variable instead of reusing `nexti`. In other place `nexti` was used as an index in `cedestr`. Here `i - op_start + 1` is the size of the instruction.

Reply:
done: I added "Py_ssize_t ilen".